packages/storage/test/requests.test.ts

```diff
@@ -172,9 +171,9 @@ describe('Firebase Storage > Requests', () => {
       [locationNormal, locationNormalUrl],
       [locationEscapes, locationEscapesUrl]
     ];
-    for (let i = 0; i < maps.length; i++) {
-      const location = maps[i][0] as Location;
-      const url = maps[i][1] as string;
+    for (const d of maps) {
```

Comment:
While you are here, you could make this:

```
    for (const [location, url ] of maps) {
```

Reply:
Done